exporter/lokiexporter/exporter.go

```diff
@@ -150,6 +178,10 @@ func (l *lokiExporter) logDataToLoki(ld pdata.Logs) (pr *logproto.PushRequest, n
 		}
 	}
 
+	if errs != nil {
+		l.logger.Error("some logs has been dropped", zap.Error(errs))
```

Comment:
Sorry if I asked this before, but how often do you think this would happen? I'm worried that this might flood the collector logs. Perhaps add a todo to convert this to a metric, once we sort out the observability for components?

Reply:
Honestly I don't have any ideas.
I would say in a correct context with compatibles receivers, never (don't use `syslogreceiver` for exemple which create a structured body without otel attributes, I have some doubts with `filelogreceiver`, I don't remember my tests :/ ).
But if it happen it will be probably a lot even if we could imagine some isolated cases related to a custom OTLP producer.

In term of monitoring, I'm almost sure we already have this metrics thanks to `numDroppedLogs` above but we don't know the reason. Maybe we should move this log to "Debug" instead of "Error" ?

Reply:
Debug instead of Error would certainly be better, especially as it would happen for all the logs for a given setup, evident when the collector first starts.